app/templates/new-home-view.jade

```diff
@@ -5,36 +5,36 @@ mixin box
     if me.isAnonymous() == true
       h6#classroom-edition-header(data-i18n="new_home.classroom_edition")
       div
-        button.teacher-btn.btn.btn-primary.btn-lg.btn-block(data-i18n="new_home.im_a_teacher")
+        button.teacher-btn.btn.btn-primary.btn-lg.btn-block(data-event-action="Homepage Click Teacher Button CTA", data-i18n="new_home.im_a_teacher")
       div
-        a.btn.btn-forest.btn-lg.btn-block(href="/courses", data-i18n="new_home.im_a_student")
+        button.student-btn.btn.btn-forest.btn-lg.btn-block(data-i18n="new_home.im_a_student")
```

Comment:
Actually, even better: Most of these use the router navigate anyway, so you can just have this and let the link work normally, just inserting the event logging:
```
  onClickStudentLink: (e) ->
    window.tracker?.trackEvent 'Homepage Click Student Button', category: 'Homepage', ['Mixpanel']
```